Tests/GauntletTests/Operators/ThrowingOperatorsTests.swift

```diff
@@ -53,7 +53,9 @@ class ThrowingAssertionsTestCase: XCTestCase {
         // Then
         Assert(that: assertion)
             .didFail(expectedName: "doesNotThrow", expectedLine: expectedLine)
-            .isEqualTo(.thrownError(MockError.someError))
+            .isThrownError()
+            .isType(MockError.self)
```

Comment:
Now that we are discussing tenses... `isType` or `ofType`?

Reply:
`ofType` makes a better sentence, but the `is` prefix makes it a pinch more discoverable. Per my comment below about combining these two, `isThrownError(ofType:)` could work.

Reply:
I also don't love `isType`, maybe `isOfType`?

Reply:
Updated the operator name to `isOfType`.